src/theme/components.js

```diff
@@ -17,6 +17,9 @@ const components = {
       variant: 'contained',
     },
     styleOverrides: {
+      contained: {
```

Comment:
I decided to apply it only for "contained" buttons. I thought that other types of buttons, if we use them, would possibly be different. Happy to move it to "root" if you prefer though.

Reply:
Also I wonder if maxHeight is better, what do you think?

Reply:
yes, maxHeight is better.

Reply:
Thanks for the feedback!
So....
I tried maxHeight, but this has some issues.

See the download button for example:
![image](https://github.com/mozilla/perfcompare/assets/454175/46765c0b-4666-44f4-b39e-6b6ce760f638)
We definitely want it to take the full height. I tried to play with flex to make it take the full height even with this maxHeight, but this wouldn't work. I'd have to override it.

If I use minHeight instead, then I'll have similar problems with the filtering options (because they'll be buttons):
![image](https://github.com/mozilla/perfcompare/assets/454175/0142232d-5c3e-4a5c-9726-554240839b9e)

Now I believe `height` is easier because it's more easily overridable. It's even overriden "by default" when using Flex, because it's used as the flex-basis and then the flex algorithm makes it grow/shrink, while max/minHeight are very strict.

So in the end I kept it with "height"!